ggshield/core/lines.py

```diff
@@ -175,25 +151,22 @@ def get_lines_from_patch(content: str, filemode: Filemode) -> Iterable[Line]:
                 post_index -= 1
                 line_pre_index = None
                 line_post_index = None
-                category = LineCategory.EMPTY
         elif line_type == "+":
             post_index += 1
             line_post_index = post_index
-            line_content = line[1:]
-            category = LineCategory.ADDITION
+            line_content = line
         elif line_type == "-":
             pre_index += 1
             line_pre_index = pre_index
-            line_content = line[1:]
-            category = LineCategory.DELETION
+            line_content = line
         elif line_type == "\\":
             # This type of line shouldn't contain any secret; no need to set indices
-            line_content = line[1:]
+            line_content = line
```

Comment:
we can probably simplify the setting of line_content, it is almost always set to line ?

Reply:
Good point. Looking into it.